eng/tools/generator/autorest/model/changelog.go

```diff
@@ -268,10 +268,23 @@ func getRemovedContent(removed *delta.Content) []string {
 	}
 	// write functions
 	if len(removed.Funcs) > 0 {
+		var lroItem []string
 		for _, k := range sortFuncItem(removed.Funcs) {
+			v := removed.Funcs[k]
+			if v.ReplacedBy != nil {
```

Comment:
out of curiosity, can we detected something is replaced by something else?

Reply:
lro to non-lro or non-lor to lro

Reply:
example: Operation `*AgentPoolsClient.AbortLatestOperation` has been changed to LRO, use `*ManagedClustersClient.BeginAbortLatestOperation` instead.

ReplacedBy save *ManagedClustersClient.BeginAbortLatestOperation